week0/homework/index.js

```diff
@@ -2,16 +2,19 @@ const Person = require("./Person.js");
 
 // remember to write a constructor function
 let jimmy = new Person("Jimmy", 28);
+console.log(jimmy);
 
-jimmy.addPhone("55551234");
+// jimmy.addPhone("55551234");
+ console.log(jimmy.addPhone("55551234"));
 
-jimmy.call();
-// should say "Calling Jimmy at 55551234..."
+// jimmy.call();
+ console.log(jimmy.call());
```

Comment:
Just watch the indentation, don't add extra spaces. The `Person.js` file from you is perfect.